modules/mods.d/mods.js

```diff
@@ -1,71 +1,69 @@
-let rp = require('request-promise');
-const {
-    createRichEmbed
-} = require('./../../discord-util.js');
+const rp = require('request-promise');
+
+const util = require('./../../discord-util.js');
+
+//update every hour
+const timer = (1 * 3600 * 1000);
+const url = 'https://raw.githubusercontent.com/CCDirectLink/CCModDB/master/mods.json';
+const CCLoaderLink = 'https://github.com/CCDirectLink/CCLoader';
+const CCModDB = 'https://github.com/CCDirectLink/CCModDB';
+
 module.exports = class Mods {
```

Comment:
As some functions of the `Mods` class use objects structured as in the [] file, it would be wise to add a typedef for this object.

```suggestion
/**
 * A mod's information, as described by the DirectLink mods database (`mods.json`).
 * @typedef {{name: string, description: string, dir: ?{"any": string}, license: ?string, page: {name: string, url: string}[], archive_link: string, hash: {type: string}, version: string}} Mod
 */

module.exports = class Mods {
```